taxi_service/settings.py

```diff
@@ -122,7 +122,11 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
```

Comment:
The `STATIC_URL` setting is correctly set to `/static/`, which is necessary for serving static files.